goldens/size-tracking/aio-payloads.json

```
@@ -3,26 +3,26 @@
     "master": {
       "uncompressed": {
         "runtime-es2015": 3033,
-        "main-es2015": 450953,
-        "polyfills-es2015": 52215
+        "main-es2015": 451829,
+        "polyfills-es2015": 55228
       }
     }
   },
   "aio-local": {
     "master": {
       "uncompressed": {
         "runtime-es2015": 3033,
-        "main-es2015": 451600,
-        "polyfills-es2015": 52215
+        "main-es2015": 452581,
+        "polyfills-es2015": 55228
       }
     }
   },
   "aio-local-viewengine": {
     "master": {
       "uncompressed": {
         "runtime-es2015": 3153,
-        "main-es2015": 437306,
-        "polyfills-es2015": 52493
+        "main-es2015": 438257,
```

```suggestion
        "main-es2015": 438598,
```

On master this value is `437622`.